functions-and-operators/tidb-functions.md

```diff
@@ -102,22 +100,11 @@ select tidb_decode_key('7480000000000000FF3E5F720400000000FF0000000601633430FF33
 1 row in set (0.001 sec)
 ```
 
-### MySQL compatibility
-
-The `TIDB_DECODE_KEY` function is TiDB-specific and not compatible with MySQL.
-
-## TIDB_DECODE_PLAN
+### TIDB_DECODE_PLAN
 
-The `TIDB_DECODE_PLAN` function can be used to decode a TiDB execution plan. These plans can be found in the slow query log.
+Plans can be found in encoded form in the slow query log. The `TIDB_DECODE_PLAN()` function is then used to reverse the plan into a human-readable form.
 
-### Synopsis
-
-```ebnf+diagram
-TableStmt ::=
-    "TIDB_DECODE_PLAN(" STR ")"
-```
-
-### Example
+This is useful because the plan is captured at the time the statement executes. Re-executing the statement in `EXPLAIN` might produce different results as data distribution and statistics will evolve over time.
```

Comment:
```suggestion
This function is useful because a plan is captured at the time the statement is executed. Re-executing the statement in `EXPLAIN` might produce different results as data distribution and statistics evolves over time.
```